docs/reference/commands/disk.rst

```diff
@@ -33,13 +33,27 @@ Global flags:
 ``add``
 -------
 
-Adds a disk to the cluster, alongside optional devices for write-ahead logging and database management.
+Adds a new Ceph disk (OSD) to the cluster, alongside optional devices for write-ahead logging and database management.
+
+The command takes a parameter SPEC which is either a path to a block device such as /dev/sdb or a specification for one or more loop files.
+
+The specification for loop files is of the form "loop,<size>,<nr>"
+
+- size is a number with M, G, or T suffixes for megabytes, gigabytes, or terabytes.
+- nr is the number of file-backed loop OSDs to create.
```

Comment:
`nr`

i think we should settle upon one way of expressing these devices. If we are qualifying the OSD then perhaps: "loopback-based OSD" or "file-backed OSD" but "file-backed loop OSD" sounds off to me